keyboard_rewrite/src/lib/y2_keyboard/clients/main.rb

```diff
@@ -0,0 +1,14 @@
+require_relative "../dialogs/layout_selector"
+
+module Y2Keyboard
+  module Clients
+    class Main
```

Comment:
np: Why `Main`? After all, when you are invoking the client, you are using `yast2 keyboard`.

Reply:
When you suggest me [this change](https://github.com/yast/yast-country/pull/144/files/0599b6009b6546ed7c0c78f20a27368778fbd2a2#diff-20cc8e98fac2471531f1ee0b6fcd12dd), I take [this]( https://github.com/yast/yast-storage-ng/blob/master/src/clients/partitioner.rb) as example where the client is also named ```Main``` 
There is no other reason, so I will rename it.